lib/active_merchant/billing/gateways/moneris.rb

```diff
@@ -438,6 +438,19 @@ def wallet_indicator(token_source)
         }[token_source]
       end
 
+      def format_order_id(wallet_indicator_code, order_id = nil)
+        # Truncate (max 100 characters) order id for
+        # google pay and apple pay (specific wallets / token sources)
+        wallets = %w(APP GPP)
+        return truncate_google_pay_order_id(order_id) if wallets.include?(wallet_indicator_code)
+
+        order_id
+      end
+
+      def truncate_google_pay_order_id(order_id = nil)
```

Comment:
The method is references `google_pay` specifically, but it seems we need to truncate if  it's AP or GP. Should we change the name of the method to just be `truncate_order_id`?

Reply:
good catch I will :)